cluster-provision/gocli/opts/rookceph/manifests/operator.yaml

```diff
@@ -600,7 +600,7 @@ spec:
       serviceAccountName: rook-ceph-system
       containers:
         - name: rook-ceph-operator
-          image: docker.io/rook/ceph:master
+          image: quay.io/kubevirtci/rook-ceph:v1.16.6
```

Comment:
will this not create problems whenever people are updating to newer versions - meaning they have to be aware that we are actually mirroring the image to our own registry?

maybe we could try something as patching the yaml with kustomize? This way we could just replace the original yaml with the newer one and the customization would be taken care of when applying it?

Reply:
This would not be straight forward as gocli applies the yamls - I am not sure we should add an extra step with kustomize. 

I could probably change the image using some regex in gocli but then the actual image that is used is  hidden away in gocli - having the image here in the manifests is clear for anyone who needs to find it.